tests/test.sh

```diff
@@ -40,38 +40,46 @@ cp -f main_server/reval/reval.js tests/backup/main_server/
 
 NUMBER_OF_EXECUTION_NODES=5
 LOGGERCONFIG='../deploy/configs/util/logger.json'
-export NUMBER_OF_EXECUTION_NODES LOGGERCONFIG
+LBCONFIG='../deploy/configs/load_balancer/nodes_data_conf.json'
+NODE_TLS_REJECT_UNAUTHORIZED=0
+MSCONFIG="../deploy/configs/main_server/conf.json"
+MSLABCONFIG="../deploy/configs/main_server/labs.json"
+MSCOURSECONFIG="../deploy/configs/main_server/course.json"
+MSAPIKEYS="../deploy/configs/main_server/APIKeys.json"
+export NUMBER_OF_EXECUTION_NODES LOGGERCONFIG LBCONFIG NODE_TLS_REJECT_UNAUTHORIZED
+export MSCONFIG MSLABCONFIG MSCOURSECONFIG MSAPIKEYS ENCONFIG ENSCORES
 # change the config file paths in all the relevant js files
-sed -i 's/\/etc\/load_balancer/\.\.\/deploy\/configs\/load_balancer/' load_balancer/load_balancer.js
-sed -i 's/\/etc\/main_server/\.\.\/deploy\/configs\/main_server/' main_server/main_server.js
+#sed -i 's/\/etc\/load_balancer/\.\.\/deploy\/configs\/load_balancer/' load_balancer/load_balancer.js
```

Comment:
please remove the next two commented lines. There are equivalent environment config variables.